Convergence/IO/EPICS/CA/ChannelAccessWrapper.cs

```diff
@@ -149,26 +145,26 @@ public static EcaType ca_context_create(PreemptiveCallbacks preemptive)
             //   ECA_NOTTHREADED - Current thread is already a member
             //                     of a non-preemptive callback CA context
             // https://epics.anl.gov/base/R3-15/9-docs/CAref.html#ca_context_create
-            
+
             // https://epics.anl.gov/base/R3-15/9-docs/CAref.html#ca_current_context
-            static extern Int32 ca_context_create(PreemptiveCallbacks p);
+            static extern int ca_context_create(PreemptiveCallbacks p);
 
         }
 
         // Needed when you invoke a 'ca_' function on a worker thread
 
-        public static IntPtr ca_current_context()
+        public static nint ca_current_context()
         {
             return new(ca_current_context());
             [DllImport(CA_DLL_NAME)]
             // https://epics.anl.gov/base/R3-15/9-docs/CAref.html#ca_current_context
-            static extern IntPtr ca_current_context();
+            static extern nint ca_current_context();
         }
 
-        public static EcaType ca_attach_context(this IntPtr context)
+        public static EcaType ca_attach_context(this nint context)
         {
             // Try Parse Enum of type <EcaType> from Int32 return by (ca_attach_context(context))
-            if (Enum.TryParse<EcaType>(CA_EXTRACT_MSG_NO(ca_attach_context(context)).ToString(), out EcaType result))
+            if (Enum.TryParse(CA_EXTRACT_MSG_NO(ca_attach_context(context)).ToString(), out EcaType result))

```

Comment:
Any specific reason to remove <EcaType>?
I guess it just provide an extra check.